ios-base/Info.plist

```diff
@@ -39,6 +39,8 @@
 	<string></string>
 	<key>FacebookKey</key>
 	<string>$(FACEBOOK_KEY)</string>
+	<key>FacebookClientToken</key>
+	<string></string>
```

Comment:
i think we have deprecated the facebook feature, so you can remove it

Reply:
Sure I'll do that in other PR to avoid extending this further